jf_agent/jf_jira/jira_download.py

```diff
@@ -1,14 +1,178 @@
+import json
 import logging
+import math
+import queue
+import random
+import re
 import string
+import sys
+import threading
+import traceback
+from collections import defaultdict, namedtuple
+from typing import Dict, Optional
 
+from dateutil import parser
 from jf_ingest import diagnostics, logging_helper
+from jf_ingest.config import IngestionConfig
+from jf_ingest.jf_jira import load_and_push_jira_to_s3
+from jf_ingest.jf_jira.auth import get_jira_connection as get_jira_connection_from_jf_ingest
+from jf_ingest.jf_jira.custom_fields import (
+    IssueJCFVDBData,
+    JCFVDBData,
+    JCFVUpdate,
+    JCFVUpdateFullPayload,
+    identify_custom_field_mismatches,
+)
 from jf_ingest.utils import retry_for_status
 from jira.exceptions import JIRAError
+from jira.resources import dict2resource
+from jira.utils import json_loads
+from tqdm import tqdm
 
+from jf_agent.config_file_reader import ValidatedConfig
 from jf_agent.jf_jira.utils import retry_for_status
+from jf_agent.util import UserProvidedCreds, split
 
 logger = logging.getLogger(__name__)
 
+MAKARA_CUSTOM_FIELD_MISMATCH_AND_DETECTION_FLAG = (
+    'makara-custom-fields-mismatch-detection-and-repair'
+)
+
+
+@diagnostics.capture_timing()
+@logging_helper.log_entry_exit(logger)
+def run_jira_download(config: ValidatedConfig, ingest_config: IngestionConfig) -> dict:
+    from jf_agent.jf_jira import print_all_jira_fields
+
+    logger.info(
+        'Running Jira Download...',
+    )
```

Comment:
Nit - is there any reason why this is multi-line?

Reply:
I think the `black` formatting commit hook did this. No reason, just styling